runtime-config-linux.md

```diff
@@ -156,11 +162,13 @@ The cgroups will be created if they don't exist.
    "cgroupsPath": "/myRuntime/myContainer"
 ```
 
-`cgroupsPath` can be used to either control the cgroups hierarchy for containers or to run a new process in an existing container.
+The container process will be added to the cgroups at `cgroupsPath`.
 
 You can configure a container's cgroups via the `resources` field of the Linux configuration.
 Do not specify `resources` unless limits have to be updated.
-For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
+For example, to run a new process in an existing container without updating limits, specify `cgroupsPath` but not `resources`.
+To run a new process in an existing container and update limits, specify both `cgroupsPath` and `resources`.
+To run a new process in a new container with explicit resource limits, specify `resources` and optionally set `cgroupsPath` to a cgroup that does not yet exist.
```

Comment:
On Wed, Jan 06, 2016 at 02:26:40AM -0800, Brandon Philips wrote:
> This won't work when launching under systemd which creates a cgroup
> before launching a process.

I'm not familiar with systemd and its interaction with OCI-specified
cgroups.  What exactly would not work?  If systemd created cgroups,
you use the path to the cgroups it will create in cgroupsPath, and
your container joins them.  If systemd does not create cgroups with
unified-hierarchy-like naming, we probably need to relax [2] and come
up with a different approach (but that's a big enough change that it
probably deserves mailing-list discussion).

> I don't understand why we are tying the concept of container to
> creation of cgroup.

I agree, but I think that's what [1] and #237 are about.  This PR is
about clarifying behavior until the spec drops cgroup handling, which
may never happen.

[1]: https://groups.google.com/a/opencontainers.org/forum/#!topic/dev/qWHoKs8Fsrk
     Subject: removal of cgroups from the OCI Linux spec
     Date: Wed, 28 Oct 2015 17:01:59 +0000
     Message-ID: <CAD2oYtO1RMCcUp52w-xXemzDTs+J6t4hS5Mm4mX+uBnVONGDfA@mail.gmail.com>
[2]: https://github.com/opencontainers/specs/pull/137/files#diff-b753d6f046da0a2956b5218efd779b7fR139